src/applications/appeals/testing/sc/content/optionForMst.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+export const optionForMstTitle =
+  'Would you like to view an option available to you if your claim is related to Military Sexual Trauma (MST)';
+
+export const supportInfo = (
+  <va-additional-info trigger="How can I find support?">
+    <div>
+      <p className="vads-u-margin-top--0">
+        <a href="/health-care/about-va-health-benefits/" target="_blank">
+          Learn more about Veterans Health Administration (VHA) health care
+          services
+        </a>
+      </p>
+      <p>
+        To learn about VHA health care services available related to MST,
+        contact a VHA MST Coordinator.
+      </p>
+      <p>
+        <a
+          href="https://www.mentalhealth.va.gov/msthome/vha-mst-coordinators.asp"
+          target="_blank"
+          rel="noreferrer"
```

Comment:
Just a head's up that DST is just about to complete a `va-link` web component variation for external link: https://github.com/department-of-veterans-affairs/component-library/pull/1269